fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonBlockTags.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.BlockTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonBlockTags {
```

Comment:
I would like to see a centralized tag for blocks that must not be moved by things that ignore piston behavior such as Carrier pickup, Create contraptions or AE2 spatial storage. For blocks such as chunk loaders or pipes, this can lead to soft or hard world corruption. I don't know if it would fit fabric API. My personal suggestion is `c:must_not_be_moved`, but `c:unmoveable_blocks` or similar would work too.

Reply:
Added. I think in this case it makes the most sense for a clear standard tag to be defined.